asm/vm/asm_vm

```diff
@@ -16,27 +16,47 @@ fi
 ### Internal variables ###
 MAJOR="${MAJOR:=1}"
 MINOR="${MINOR:=8}"
-POINT="${POINT:=1}"
+POINT="${POINT:=2}"
 ASM_RELEASE="${MAJOR}.${MINOR}"
 ASM_META_VERSION="${MAJOR}.${MINOR}.${POINT}"
 readonly ASM_RELEASE
 readonly ASM_META_VERSION
 
+SERVICE_PROXY_AGENT_BUCKET=\
+"gs://gce-mesh/service-proxy-agent/releases/service-proxy-agent-0.2-asm.tgz"
+readonly SERVICE_PROXY_AGENT_BUCKET
+ASM_REVISION_LABEL_KEY="istio.io/rev"
+readonly ASM_REVISION_LABEL_KEY
+EXPANSION_GATEWAY_NAME="istio-eastwestgateway"
+readonly EXPANSION_GATEWAY_NAME
+ISTIOD_NAME="istiod"
+readonly ISTIOD_NAME
+IDENTITY_PROVIDER_ANNOTATION_KEY="security.cloud.google.com/IdentityProvider"
+readonly IDENTITY_PROVIDER_ANNOTATION_KEY
+
 SCRIPT_NAME="${0##*/}"
 
 PROJECT_NUMBER=""
 
-DNS_IP=""
+### Cluster variables ###
 ROOT_CERT=""
 INGRESS_IP=""
-MESH_AGENT_BUCKET=\
-"gs://gce-mesh/service-proxy-agent/releases/service-proxy-agent-0.2-asm.tgz"
-INSTANCE_IPS=""
+ASM_REVISIONS=""
+NETWORK=""
+MEMBERSHIP_UID=""
+GKE_URL=""
+ENVIRON_PROJECT_ID=""
+ENVIRON_MEMBERSHIP_ID=""
```

Comment:
Please change back to HUB for this PR re: other comment thread

Reply:
I am removing this membership id variable. This is not needed by VM script.